README.md

```diff
@@ -83,7 +83,7 @@ aicommit config set <key>=<value>
 | `OPENAI_KEY`        | N/A             | The OpenAI API key.                                                                   |
 | `locale`            | `en`            | Locale for the generated commit messages.                                             |
 | `generate`          | `1`             | Number of commit messages to generate.                                                |
-| `model`             | `gpt-3.5-turbo` | The Chat Completions model to use.                                                    |
+| `model`             | `gpt-4o-mini`   | The Chat Completions model to use.                                                    |
```

Comment:
@PeterDaveHello I think this model should be `gpt-4o` according to the [current supported models](https://github.com/openai/tiktoken/blob/c0ba74c238d18b4824c25f3c27fc8698055b9a76/tiktoken/model.py#L22)

Reply:
I don't understand. It's quite clear:

> As of July 2024, gpt-4o-mini should be used in place of gpt-3.5-turbo, as it is cheaper, more capable, multimodal, and just as fast. gpt-3.5-turbo is still available for use in the API.

https://platform.openai.com/docs/models/gpt-3-5-turbo

Reply:
I completely agree with you - and i wish i could use the `-mini` , but as far as i can see it's only the `-4o` that is supported right now in their dictionary ?

Let me know if i'm misunderstanding something :)

Reply:
Is there any dependency on the repo you mentioned? I don't know why it's relevant.

Reply:
Yes - it's dependant on **OpenAI's** TikToken repo here: https://github.com/openai/tiktoken/blob/c0ba74c238d18b4824c25f3c27fc8698055b9a76/tiktoken/model.py#L22

Reply:
This TypeScript project depends on a Python repo?

Reply:
Maybe this is what you mean:
https://github.com/negoziator/ai-commit/blob/cad5e7967c418e23945d49fa463b2c7a19ef2ddb/package.json#L32

Then we should wait for https://github.com/dqbd/tiktoken/pull/112. Otherwise, I don't understand why you asked me to open a PR here.

Reply:
You are absolutely right 👍🏼

Reply:
Maybe using the official `openai-node` lib will prevent similar issue in the feature because they already supported `gpt-4o-mini`?

https://github.com/openai/openai-node/pull/942

Reply:
@PeterDaveHello Good point. I will remove the dependency as it gives no value 😄

Reply:
If you update your branch now - i think it should work :)